scripts/bz.py

```diff
@@ -15,14 +21,10 @@
 BZ_ISLINUX = BZ_OS == 'linux'
 BZ_ISDARWIN = BZ_OS == 'darwin'
 BZ_ISWIN = BZ_OS == 'windows'
-BZ_ISVERBOSE = '--verbose' in sys.argv
 
-def get_arch(short_names = True, use_mac_rosetta = False):
+def get_arch(short_names = True):
     arch = platform.machine().lower()
 
-    if BZ_ISDARWIN and arch == 'arm64' and use_mac_rosetta:
```

Comment:
Hope you've mentioned line 23. Ah, initially we used that line to generate a `x64` binary on `arm64` systems since we didn't generate a specific `arm64` binary